Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -383,6 +384,8 @@ public override void Initialize()
 
                 "create table if not exists " + ChaptersTableName + " (ItemId GUID, ChapterIndex INT NOT NULL, StartPositionTicks BIGINT NOT NULL, Name TEXT, ImagePath TEXT, PRIMARY KEY (ItemId, ChapterIndex))",
 
+                "create table if not exists " + TrickplayTableName + " (ItemId GUID, Width INT NOT NULL, Height INT NOT NULL, TileWidth INT NOT NULL, TileHeight INT NOT NULL, TileCount INT NOT NULL, Interval INT NOT NULL, Bandwidth INT NOT NULL, PRIMARY KEY (ItemId, Width))",
```

Comment:
I feel we should be using EF for this especially if there's no foreign key.

Reply:
Can we just use EF on a non-EF database?

Reply:
In theory yes, but it should be placed in the jellyfin.db